src/index.ts

```diff
@@ -32,8 +33,26 @@ import { Chain } from './chainList'
 import Popup from './popup'
 import { ModalController } from './ui/modalController'
 
+function preLoadIframe(url: string, appId: string) {
+  try {
+    if (typeof document === 'undefined') return
+    const iframeLink = document.createElement('link')
+    iframeLink.href = `${url}/${appId}/login`
+    iframeLink.type = 'text/html'
+    iframeLink.rel = 'prefetch'
+    if (iframeLink.relList && iframeLink.relList.supports) {
```

Comment:
This condition may be skipped. The [DOMTokenList API](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList#browser_compatibility) has full support across all browsers.

Reply:
Removed.